packages/amplication-design-system/src/components/SelectField/SelectField.scss

```diff
@@ -13,6 +13,10 @@
 
   .select-field__container {
     font-size: var(--normal-font-size);
+
+    input {
+      color: white !important;
```

Comment:
Why did you use `important`?

Reply:
By default, the input field ``` inherits``` the text-color  as its default element style, and hence the ```important``` was required to make sure the select field input has ```color: white```

Reply:
Can you overcome the need of `!important` by being more specific, rather then using a tag selector (`input {}`)?

Reply:
@KushalRaut any updates?

Reply:
I tried many ways to make sure that the input field text becomes white but most of them didn't work and the ```!important``` tag is necessary because the input field has inline styling of ```color: inherit```